src/Symfony/Component/Config/Definition/ArrayNode.php

```diff
@@ -209,8 +209,8 @@ public function getDefaultValue()
      */
     public function addChild(NodeInterface $node)
     {
-        $name = $node->getName();
-        if (!strlen($name)) {
+        $name = (string) $node->getName();
```

Comment:
Ok, shall I also revert then changes in Symfony/Component/Security/Acl/Domain/DoctrineAclCache.php, as I added a scalar type check?